projects/developer/src/assets/appConfig.json

```diff
@@ -14,5 +14,7 @@
     "secondaryLightColor": "#5c97bf",
     "secondaryDarkColor": "#24567f",
     "logoImageCss": "",
-    "logoImage": ""
+    "logoImage": "",
+    "bannerMessage": "We are experiencing a temporary outage. ",
```

Comment:
hmm i think we should leave these empty? and just document these values as examples in the readme?

Reply:
Yes, the only reason I left something there was so that you could actually see the banner for testing.  The default should definitely be empty (no banner).